exchanges/stream/websocket.go

```diff
@@ -478,7 +482,7 @@ func (w *Websocket) Shutdown() error {
 }
 
 // FlushChannels flushes channel subscriptions when there is a pair/asset change
-func (w *Websocket) FlushChannels() error {
+func (w *Websocket) FlushChannels(ctx context.Context, allowAutoSubscribe bool) error {
 	if !w.IsEnabled() {
 		return fmt.Errorf("%s websocket: service not enabled", w.exchangeName)
 	}
```

Comment:
What about when w.features.Subscribe is true? Same for `FullPayloadSubscribe`? If you call this function and `allowAutoSubscribe` is `false`, but `w.features.Subscribe` or `FullPayloadSubscribe` is `true`, then it will generate subscriptions and subscribe?
For reference, there are 21 exchanges that will not use `allowAutoSubscribe` because `Subscribe` is `true` and 1 exchange with `FullPayloadSubscribe`

This may be intentional, but its a little confusing unless you are manually modifying `Features` externally.

Reply:
> This may be intentional
Probably not 😆 I removed this bool.